components/header/TopBar.tsx

```diff
@@ -0,0 +1,99 @@
+import Image from "apps/website/components/Image.tsx";
+import { Logo } from "../../components/header/Header.tsx";
+
+// Make it sure to render it on the server only. DO NOT render it on an island
+function TopBar({
+  logoBrandili,
+  logoMundi,
+  bgColor,
+  device,
+}: {
+  logoBrandili?: Logo;
+  logoMundi?: Logo;
+  device: "mobile" | "desktop" | "tablet";
+  bgColor?: "bg-red-0" | "bg-gold-1";
+}) {
+  // Mobile header
+  if (device === "mobile") {
+    return (
```

Comment:
A condicional pode ser adicionada ao return do componente com ternário, ficando dessa forma:
```suggestion
   return (device === "mobile") ? (
```

Reply:
corrigido